README.md

```diff
@@ -59,6 +59,10 @@ Drush alias for the **current** Silta feature branch deployment is `drush @curre
 - `lando varnishadm <commands>` - run [varnishadm](https://varnish-cache.org/docs/6.0/reference/varnishadm.html) commands.
 - `lando xdebug <mode>` - load [Xdebug](https://xdebug.org/) in the selected [mode(s)](https://xdebug.org/docs/all_settings#mode).
 
+#### Secrets handling
+
+Perform encryption and decryption operations by using [Silta CLI](https://github.com/wunderio/silta-cli). Use `silta secrets -h` to see the available commands. Use the `SECRET_KEY` value of the corresponding Silta context environment variable as the password.
```

Comment:
Should it be mentioned that SECRET_KEY can be found from the password manager (LastPass currently)?

Reply:
The up-to-date one is always available via the corresponding Silta context. This applies even more when we switch to per-project `SECRET_KEY` approach.

Reply:
Variable name too can be anything as long as it is consistent.
For example
`            - silta/decrypt-files:
                files: silta/silta-prod-secret.yaml
                secret_key_env: MY_SECRET_NAME
`